containers/datalab/content/setup-env.sh

```diff
@@ -15,12 +15,14 @@
 
 # Sets up various environment variables within the docker container.
 
-export DATALAB_USER=`gcloud -q config list --format yaml | grep account | awk -F" " '{print $2}'`
-export DATALAB_PROJECT_ID=`gcloud -q config list --format yaml | grep project | awk -F" " '{print $2}'`
-if [ -z $DATALAB_PROJECT_NUM ]; then
-  export DATALAB_PROJECT_NUM=`curl --silent -H "Metadata-Flavor=Google" http://metadata.google.internal/computeMetadata/v1beta1/project/numeric-project-id`
+if [ -z "$GOOGLE_APPLICATION_CREDENTIALS" ] && [ -z "$GCLOUDSDK_CONFIG" ]
+then
+  export CLOUDSDK_CONFIG=/content/datalab
 fi
-if [ -z $DATALAB_INSTANCE_NAME ]; then
-  export DATALAB_INSTANCE_NAME=$GAE_MODULE_VERSION
+if [ -z "$DATALAB_ENV" ]
+then
+  export DATALAB_ENV="local"
 fi
 
+export DATALAB_USER=`gcloud -q config list --format yaml | grep account | awk -F" " '{print $2}'`
```

Comment:
Is this still needed? The only usage of the variable seems to be as home dir name for jupyter. Since local run is no longer for multiple users, it might be Okay to default it to "anonymous" as in https://github.com/GoogleCloudPlatform/datalab/blob/4a86c50c0a1f366ad2239e3d156780659824abaf/sources/web/datalab/userManager.ts#L51.